lib/exrabbit/util.ex

```diff
@@ -87,6 +87,9 @@ defmodule Exrabbit.Util do
   end
 
   defp get_formatter_module(atom) do
-    Module.concat([Exrabbit.Formatter, Atom.to_string(atom) |> String.upcase()])
```

Comment:
Protocols work in a similar manner: you get a `ProtocolName.TypeName.beam` in your `ebin/` directory for each implementation of `ProtocolName` in your app. We don't need a protocol here, so we're using a behaviour with a naming convention that is already used in ExDoc.

This convention also encourages reusing formatters: instead of having `AppFoo.ExRabbitFormatter.Msgpack` and `AppBar.BlahBlah.MsgPack`, one for each app, you can have an `Exrabbit.Formatter.MSGPACK` that can be published on its own and feel more natural to be reused in multiple apps.